src/components/media/simpleImage.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SbEditable from "storyblok-react";
+import WidthBox from "../layout/widthBox";
+import RichTextRenderer from "../../utilities/richTextRenderer";
+
+const SimpleImage = ({
+  blok: { image: { filename } = {}, imageWidth, caption, captionAlign },
+  blok,
+}) => (
+  <SbEditable content={blok}>
+    <WidthBox width={imageWidth}>
+      <figure>
+        <img src={filename} alt="" />
+        {caption && (
+          <figcaption>
+            <RichTextRenderer
+              wysiwyg={caption}
+              className={`su-caption su-text-${captionAlign}`}
```

Comment:
```suggestion
              className={`su-caption su-mt-08em su-text-${captionAlign}`}
```
Adding some top margin to the caption (editing free hand so let's see how that looks - might need adjustment) 😃